packages/ui/components/User/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { View } from 'react-native'
+import { observer } from 'startupjs'
+import propTypes from 'prop-types'
+import Avatar from '../Avatar'
+import Div from '../Div'
+import Span from '../Span'
+import './index.styl'
+
+const descriptionSizes = {
+  xxl: 'l',
+  xl: 'm',
+  l: 's',
+  m: 'xs',
+  s: 'xs',
+  xs: 'xs'
+}
+
+function User ({
+  avatarUrl,
+  backgroundColor,
+  description,
+  name,
+  avatarPosition,
+  size,
+  status,
+  onPress
+}) {
+  const extraAvatarStyles = { 'without-label': !name && !description }
+
+  return pug`
+    Div.root(
+      styleName=[avatarPosition]
+      backgroundColor=backgroundColor
+      onPress=onPress
+    )
+      Avatar.avatar(
+        styleName=[avatarPosition, extraAvatarStyles]
+        size=size
+        status=status
+        url=avatarUrl
+        fallback=name
+      )
+      View.userInfo
```

Comment:
Span.userInfo(styleName=[avatarPosition])

index.styl
.userInfo
  &.right
    text-align right

and remove

.name
.description
  &.right
     text-align right